src/helpers/AccountHelpers.ts

```diff
@@ -11,21 +11,35 @@ export const executeUpdateAccountTransactions = async (
     transactionBytes: string
 ): Promise<void> => {
     if (updateAccountTransactionBytes) {
-        const buffer = Buffer.from(updateAccountTransactionBytes, "base64");
-        const transaction = await Transaction.fromBytes(buffer).sign(privateKey);
-        const response = await transaction.execute(client);
-
-        const receipt = await response.getReceipt(client);
-        if (receipt.status !== Status.Success) {
-            throw new Error("UpdateAccountTransaction failed");
+        let attemptsLeft = 3;
+        while (attemptsLeft-- > 0) {
+            const buffer = Buffer.from(updateAccountTransactionBytes, "base64");
```

Comment:
There is obviously a risk when transaction will be successful and only receipt will not be received. So instead of redoing only receipt, you will redo a transaction that will obviously fail all next attempts and all precess will be interrupted instead of just retrying receipt.